kani-driver/src/call_cbmc.rs

```diff
@@ -103,6 +109,10 @@ impl KaniSession {
 
         args.push("--slice-formula".into());
 
+        if self.args.output_format != OutputFormat::Old {
+            args.push("--json-ui".into());
+        }
```

Comment:
Does this impact `--visualize`?

Reply:
Thanks for pointing this out. It wasn't clear to me: `--visualize` still works, but I don't know if it's adding more files. So I reverted this change and added a comment.